src/sentry/templates/sentry/emails/digests/body.html

```diff
@@ -33,7 +33,7 @@
                       {% url 'sentry-group' group.organization.slug group.project.slug group.id as group_link %}
                       <a href="{% absolute_uri group_link %}">{{ group.message_short }}</a>
```

Comment:
It might make sense to soft break this also?

We should probably soft break anywhere someone might dump a Java/Scala path.